### DISCUSSION TITLE
Issue : CANNOT LINK EXECUTABLE "node": cannot locate symbol "__emutls_get_address" referenced by "/data/data/com.termux/files/usr/bin/node"...

### DISCUSSION THREAD

**Discussion by ganeshkamal-ar** [termux/termux-app#2388] [Q&A] [answered]
I got this issue in npm "CANNOT LINK EXECUTABLE "node": cannot locate symbol "__emutls_get_address" referenced by "/data/data/com.termux/files/usr/bin/node"..."
anyone can help me please.

**Comment by agnostic-apollo** [comment, marked as answer]
Run `pkg upgrade`

https://github.com/termux/termux-packages/issues/7945
